### DISCUSSION TITLE
Moving issues to discussions?

### DISCUSSION THREAD

**Discussion by danielfdickinson** [adityatelange/hugo-PaperMod#504] [Q&A]
Is there a way to move an issue to discussions? There are enough folks not used to having 'Discussions' available that will post an issue that is more of a support / Q & A, and "it'd be nice" to be able to move the issue to discussions. Is there a way to do that?

**Comment by danielfdickinson** [comment]
I saw that the maintainer can move issues to discussions, but I'm having trouble closing this discussion. Is there a way to do that?

**Reply by adityatelange** [reply]
No way to close this discussion. Although maintainer can  delete it

**Reply by danielfdickinson** [reply]
Thank you.
